src/revocation.rs

```diff
@@ -274,4 +309,45 @@ mod tests {
         let outputs = run_revocation_actions(json, &test_config.secure_size);
         assert!(outputs.is_err());
     }
+
+    #[test]
+    fn get_revocation_cert_path_default() {
+        let test_config = KeylimeConfig::default();
+        let revocation_cert_path =
+            get_revocation_cert_path(&test_config).unwrap();
+        let mut expected = PathBuf::from(WORK_DIR);
+        expected.push("secure/unzipped/");
+        expected.push(REV_CERT);
+        assert_eq!(*revocation_cert_path, expected);
+    }
+
+    #[test]
+    fn get_revocation_cert_path_absolute() {
+        let mut test_config = KeylimeConfig::default();
+        test_config.revocation_cert = String::from("/test/cert.crt");
```

Comment:
I looked into the `clippy` error on this that is failing CI. If you initialize this way instead, it should get rid of the error:

```
let test_config = KeylimeConfig {
            revocation_cert: String::from("/test/cert.crt"),
            ..Default::default()
        };
```
And of course run `cargo fmt` after!

Reply:
OK, I'll modify as suggested.